tabs/configuration.js

```diff
@@ -456,6 +456,10 @@ TABS.configuration.initialize = function (callback, scrollPosition) {
             serialRXtypes.push('Spektrum Bidir SRXL');
         }
 
+        if (semver.gte(CONFIG.flightControllerVersion, "3.2.0"))  {
```

Comment:
Sure -- but I'm unclear on what apiVersion though. Best guess is it's the MSP version -- is that true? Seems odd though since this change isn't really related to MSP. Is that version kept in sync between cf/bf? 

Looks like CF code handles this incompatibility with checks like "if boardID = clfl && flightControllerVersion >= 2.0.0 or if boardID = btfl && flightControllerVersion >= 3.2.0)"